src/components/Menus/Sidebar/SidebarBar.jsx

```diff
@@ -25,7 +25,10 @@ const SidebarBar = (props) => {
   const isMobile = useMediaQuery({ query: MOBILE_MEDIA_QUERY });
 
   const expandPopOut = () => {
-    const option = instructions ? "instructions" : "file";
+    let option = "file";
+    if (instructions.length > 0) {
+      option = "instructions";
+    }
```

Comment:
you could make this a one liner again with `const option = instructions.length > 0 ? "instructions" : "file"` but happy with it either way 😄